internal/services/springcloud/spring_cloud_certificate_resource.go

```diff
@@ -105,7 +106,7 @@ func resourceSpringCloudCertificateCreate(d *pluginsdk.ResourceData, meta interf
 			return err
 		}
 		cert.Properties = &appplatform.KeyVaultCertificateProperties{
-			VaultURI:         &keyVaultCertificateId.KeyVaultBaseUrl,
+			VaultURI:         utils.String(strings.TrimSuffix(keyVaultCertificateId.KeyVaultBaseUrl, "/")),
```

Comment:
could you explain this change?

Reply:
Yes, sorry that I forgot to add it in the description.

The backend api will reject if vault uri has a suffix `/`

Reply:
and the stray `/` is coming from where?

Reply:
It was added here: https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/keyvault/parse/nested_item.go#L107